model/dataset_split.py

```diff
@@ -3,45 +3,10 @@
 
```

Comment:
we plan to not use the dataset_split here, so please remove this file from your PR, thanks!